sanic/sanic.py

```diff
@@ -348,9 +348,8 @@ def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
         log.debug(self.config.LOGO)
 
         # Serve
-        if ssl is None:
-            proto = "http"
-        else:
+        proto = "http"
```

Comment:
The else statement was not unnecessary, it's a matter of coding style. As there is's no way to enforce one coding style over the other, I am not sure we should encourage such code change, it only makes PR longer to review and sometimes puzzling. Any way, it's a minor details :)